### ISSUE TITLE
Awkward sentence

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/standard#456]
From the glossary:
```
 Revisions of code can be compared with each other and it makes it easy to restore previous versions.
```
It's not clear what the "it" in the 2nd half of this sentence refers to.

**Comment by Ainali** [comment]
How about rewriting it to:

> Version control makes it possible to compare revisions of code with each other, and the visual difference between revisions helps when restoring a previous version.

**Comment by Ainali** [comment]
Fixed by #525
